timeseries/time_series.go

```diff
@@ -35,6 +35,38 @@ func (ts *TimeSeries) AddCandle(c *Candle) error {
 	return ErrUnexpectedTime
 }
 
+// RemoveCandles remove the candle from series
+// [startIndex] is mandatory
+// [endIndex] is optional
+// returns a new timeseries object
+func (ts *TimeSeries) RemoveCandles(startIndex int, endIndex *int) (*TimeSeries, error) {
```

Comment:
Thanks for PR! It looks like useful function
What do you think to change func name to `Trim`? It's short and clear

Reply:
Sounds good